src/main/scala/com/lunatech/lunabot/LunabotServlet.scala

```diff
@@ -52,31 +52,39 @@ class LunabotServlet(val tokens: Map[Int, String]) extends ScalatraServlet with
     val hipChatMessage: HipChatMessage = jsonValue.extract[HipChatMessage]
 
     // Expression to send to the REPL
-    val cmd: String = hipChatMessage.item.message.message.replace("/scala ", "")
-    logger.debug(s"Processing $cmd")
-
-    // Response from the REPL
-    lazy val triedResponse: Try[JValue] = evaluateExpression(cmd)
-
-    // Prepare POST request
-    val maybeResponse: Option[URL] = responseUrl(hipChatMessage.item.room.id)
-    val maybeResult: Either[String, (URL, JValue)] = maybeResponse match {
-      case Some(url) =>
-        triedResponse match {
-          case Success(value) => Right(url, value)
-          case Failure(ex) =>
-            Right(url, responseJValue(s"Sorry your scala code has failed with ${ex.getMessage}", "red"))
+    val msg: String = hipChatMessage.item.message.message
+    val cmd: Option[String] = if(msg.toLowerCase.startsWith("/scala")) Some(msg.substring(7)) else None
```

Comment:
Agree, I thought it space should not be included to avoid wrong interpretation of bare "/scala" input, but I'm wrong.